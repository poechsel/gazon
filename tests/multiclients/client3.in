login u3
pass p3
whoami
w
