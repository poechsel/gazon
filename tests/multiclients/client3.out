u3
u1 u3
